extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/attribute/RequestPathAttribute.java

```diff
@@ -5,6 +5,7 @@
 public class RequestPathAttribute implements ExchangeAttribute {
 
     public static final String REQUEST_PATH = "%{REQUEST_PATH}";
+    public static final String REQUEST_PATH_SHORT = "%{R}";
```

Comment:
@geoand should be `= "%R";` for consistency

Reply:
Yeah, you are right. Thanks for pointing it out